src/meta/src/hummock/manager/time_travel.rs

```diff
@@ -388,35 +412,23 @@ impl HummockManager {
             Ok(count)
         }
 
-        let epoch = delta.visible_table_committed_epoch();
-        let version_id: u64 = delta.id.to_u64();
-        let m = hummock_epoch_to_version::ActiveModel {
-            epoch: Set(epoch.try_into().unwrap()),
-            version_id: Set(version_id.try_into().unwrap()),
-        };
-        hummock_epoch_to_version::Entity::insert(m)
-            .on_conflict(
-                OnConflict::column(hummock_epoch_to_version::Column::Epoch)
-                    // The existing row must be inserted by the common committed epoch of created MVs.
-                    // While any duplicate row must be inserted by MVs still in creation.
-                    // So the row shouldn't be updated.
-                    .do_nothing()
-                    .to_owned(),
-            )
-            .do_nothing()
-            .exec(txn)
-            .await?;
+        for (table_id, cg_id) in tables_to_commit {
+            if !select_groups.contains(cg_id) {
+                continue;
+            }
+            let version_id: u64 = delta.id.to_u64();
+            let m = hummock_epoch_to_version::ActiveModel {
+                epoch: Set(committed_epoch.try_into().unwrap()),
+                table_id: Set(table_id.table_id.into()),
+                version_id: Set(version_id.try_into().unwrap()),
+            };
+            // There should be no conflict rows.
+            hummock_epoch_to_version::Entity::insert(m)
```

Comment:
Not relevant to the purpose of this PR, but should we wrap this in a transaction? Otherwise if the later writing hummock version delta fails, the `hummock_epoch_to_version` will be inconsistent to the hummock version.

Reply:
It is in the same txn with the one in commit_epoch, `exec(txn)`